bin/parse_cert

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from __future__ import print_function
```

Comment:
For my own edification, apart from 3.x compat, what's the value add of importing the 3.x print function?